### PR TITLE
[ui] for bulk upload with service call and validations, linked form as modal, returned response modals for crud operation of employees.

### LINKED ISSUE #56: [Register] UI to upload List of Employees
Linked to : [https://github.com/cybergroupdevs/ARMS/issues/50](url)

As a user, I should be able to upload a list of employees from a .csv to the ARMS system so that I can update my employee list.

Acceptance Criteria:
1. Upload file successfully
2. Necessary validation messages for failures/successes
3. View the uploaded list in a table structure (Name, Designation, Tech Skill, Yrs of experience)
4. Apply paginations, sort, and search.
5. Users should be able to perform all CRUD operations with the data available.

### PR DESCRIPTION
**Pull request by nangchetnamaw**
Fixes #56

### CODE REVIEW

**Comment by sonalichawla16** [comment]
Resolve conflicts.

File: api/storage/csv/employee.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 employeeId,email,name,designation,role
-111,fiorucci@gmail.com,Fiorucci,intern,interviewer
-,,,,
-,,,,
-,,,,
+136,"anna@cygrp,com",Anna,intern,interviewer
+137,banana@cygrp.com,Banana,intern,interviewer
+138,cheese@cygrp.com,Cheese,intern,interviewer
```
Comment (sonalichawla16) [inline api/storage/csv/employee.csv:4]: what is this ??

File: web/src/app/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<div>hello</div>
```
Comment (sonalichawla16) [inline web/src/app/dashboard/dashboard.component.html:1]: Hello ???

File: api/storage/csv/employee.csv (hunk 1 above)
Comment (nangchetnamaw) [reply api/storage/csv/employee.csv:4]: The uploaded csv file

Comment (nangchetnamaw) [reply api/storage/csv/employee.csv:4]: Sorry, was checking csv upload. Uploaded proper names now.

File: web/src/app/dashboard/dashboard.component.html (hunk 2 above)
Comment (nangchetnamaw) [reply web/src/app/dashboard/dashboard.component.html:1]: removed

File: web/src/app/list/list.component.scss [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+	*/
+	// td:nth-of-type(1):before { content: "User"; }
+	// td:nth-of-type(2):before { content: "Last Name"; }
+	// td:nth-of-type(3):before { content: "Name"; }
+	// td:nth-of-type(4):before { content: "Designation"; }
+	// td:nth-of-type(5):before { content: "Wars of Trek?"; }
+	// td:nth-of-type(6):before { content: "Secret Alias"; }
+	// td:nth-of-type(7):before { content: "Date of Birth"; }
+	// td:nth-of-type(8):before { content: "Dream Vacation City"; }
+	// td:nth-of-type(9):before { content: "GPA"; }
```
[the remark this answers is not in the document]
Comment (nangchetnamaw) [reply web/src/app/list/list.component.scss:69]: Will consult and remove

**Pull request merged**